### PR TITLE
feat(support): Added bug report button with a compose template

### PR DESCRIPTION
**Pull request by castaway**

### CODE REVIEW

**Review by tadzik — APPROVED** [review]
Looks good :)

I'd consider abstracting the https://github.com/runbox/runbox7/pull/951/files#diff-8d554c98f4c32fee762d6a18e8e189f8898124312ba390bdc51fadd38345674fR132 bit into its own component (`RunboxNavListItem`?) since we seem to repeat that sequence quite a bit, including in [other apps](https://github.com/runbox/runbox7/blob/master/src/app/contacts-app/contacts-app.component.html#L35)

File: src/app/app.component.html [hunk 1]
```diff
@@ -129,6 +129,13 @@
       </mat-list-item>
     </mat-nav-list>
 
+    <mat-nav-list>
+      <mat-list-item (click)="composeBugReport()" matTooltip="Compose an email to runbox support">
```
Comment (tadzik) [inline src/app/app.component.html:133]: Is there any reason this is its own nav-list, rather than another list-item in the previous nav-list?

Comment (castaway) [reply src/app/app.component.html:133]: Only cos that one had an ngIf attached, I guess we could move that to the nav-list-item

**Review by gtandersen — APPROVED** [review]
Nice work @castaway!

**Pull request merged**